### DISCUSSION TITLE
Adding Columns to Table in Cassandra Database

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#733] [General Discussion]
Hello Team,

We need to enhance the sunbird.location table in our Cassandra database to store additional details related to schools. Currently, the table includes the UDISE code and school name. We need to add columns for State, District, Block, and Cluster.

**Requirement:**
On searching for a code (UDISE code), the details related to the school should be displayed. The details should include State, District, Block, and Cluster.

**Current Schema:**
The current schema of the 'sunbird.location' table includes columns such as id, name (School name), code  (UDISE code), type, parent_id.

We need to add the following columns to the 'sunbird.location'  table:
1. State
2. District
3. Block
4. Cluster

**Questions:**
1. Can we directly add these columns to the 'sunbird.location' table?
2. The feasibility of adding these columns directly to the 'sunbird.location' table.
3. Is there any other table where we need to add location details, or is 'sunbird.location' the appropriate table for this information?
4. Are there any implications or best practices we need to consider when modifying the schema of an existing table in Cassandra
5. Confirmation if this is the best approach for storing additional location details or if there is a more appropriate table for this purpose.

Thank you for your assistance.

cc: @vinukumar-vs ,@rajeevsathish, @AmiableAnil , @Krishnaj20, @chitranshu-keshav, @SanthoshVasabhaktula, @gandham-santhosh , @coolbung @komalm @manishakapadnis

**Comment by HarishGangula** [comment]
@siddhishinde0723 This is belongs sunbird obsrv, please raise this issue in https://github.com/orgs/Sunbird-Obsrv/discussions
